### PR TITLE
Add plugin to check for hidden processes (DKOM)

### PR DESCRIPTION
**Pull request by skvl**
This is the first and simplest implementation of the plugin. It checks that current process is self-cycled.

Thus if rootkit would hide two sequential processes then the check does not notice this.

### CODE REVIEW

**Comment by skvl** [comment]
To test one could use https://github.com/landhb/HideProcess

File: src/plugins/dkommon/dkommon.cpp [hunk 1]
```diff
@@ -0,0 +1,200 @@
+/*********************IMPORTANT DRAKVUF LICENSE TERMS***********************
+*                                                                         *
+* DRAKVUF (C) 2014-2017 Tamas K Lengyel.                                  *
```
Comment (tklengyel) [inline src/plugins/dkommon/dkommon.cpp]: 2019

File: src/plugins/dkommon/dkommon.cpp [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    {
+        case OUTPUT_CSV:
+            printf("dkommon," FORMAT_TIMEVAL ",%" PRIu32 ",0x%" PRIx64 ",\"%s\",%" PRIi64 "\n",
+                   UNPACK_TIMEVAL(info->timestamp), info->vcpu, info->regs->cr3, info->proc_data.name, info->proc_data.userid);
+            break;
+        case OUTPUT_KV:
+            printf("dkommon Time=" FORMAT_TIMEVAL ",PID=%d,PPID=%d,ProcessName=\"%s\"\n",
+                   UNPACK_TIMEVAL(info->timestamp), info->proc_data.pid, info->proc_data.ppid, info->proc_data.name);
+            break;
+        case OUTPUT_DEFAULT:
```
Comment (tklengyel) [inline src/plugins/dkommon/dkommon.cpp]: json output missing

File: src/plugins/dkommon/dkommon.h [hunk 3]
```diff
@@ -0,0 +1,132 @@
+/*********************IMPORTANT DRAKVUF LICENSE TERMS***********************
+ *                                                                         *
+ * DRAKVUF (C) 2014-2017 Tamas K Lengyel.                                  *
```
Comment (tklengyel) [inline src/plugins/dkommon/dkommon.h]: 2019

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: src/plugins/dkommon/dkommon.h (hunk 3 above)
Comment (skvl) [reply src/plugins/dkommon/dkommon.h]: Done!

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Pull request merged**